dist/configs/config/server.cfg

```diff
@@ -1,21 +1,21 @@
 // Example Unvanquished dedicated server config
 
 // advertise the server on public server list
-// 0 - advertise everything
-// 1 - don't advertise but reply to status queries
-// 2 - don't reply to status queries but accept connections
-// 3 - only accept LAN connections
+// 0 - advertise everything - this will make the server viewable to everyone looking at the list
+// 1 - don't advertise but reply to status queries - this allows players to join and see what map is being played, how many people are playing etc. 
```

Comment:
The correct typography for _et cætera_ abbrevation is:

```
[…] playing, etc.
```

Because it's part of the enumeration.

So this looks better this way:

```diff
- // 1 - don't advertise but reply to status queries - this allows players to join and see what map is being played, how many people are playing etc.
+ // 1 - don't advertise but reply to status queries, this allows players to join and see what map is being played, how many people are playing, etc.
```